Resources/Prototypes/Entities/Structures/Piping/Atmospherics/unary.yml

```diff
@@ -17,7 +17,7 @@
           pipeDirection: South
     - type: CollideOnAnchor
     - type: StaticPrice
-      price: 200
+      price: 150
```

Comment:
These prices got dumped in my PR so revert, these are extremely easy to spam out.

Reply:
done